psydac/api/tests/test_2d_navier_stokes.py

```diff
@@ -135,9 +152,7 @@ def run_time_dependent_navier_stokes_2d(filename, dt_h, nt, newton_tol=1e-4, max
     l2norm_dp_h = discretize(l2norm_dp, domain_h, V2h, backend=PSYDAC_BACKEND_GPYCCEL)
 
     # compute the initial solution
-    equation_stokes_h.set_solver('bicg', tol=1e-15)
-    x0 = equation_stokes_h.solve()
-
+    x0 = equation_stokes_h.solve(solver='bicg', tol=1e-15)
```

Comment:
I thought that this could not work anymore, since we have changed the interface...

Reply:
ah yes, it might be using the cg, I will change that

Reply:
I have fixed it in [3d79d5f](https://github.com/pyccel/psydac/pull/184/commits/3d79d5f4cd7c85bebfb79f268fc6c64ea5c9ae4f)